scripts/plotting/polar_map.py

```diff
@@ -291,27 +301,33 @@ def polar_map(adfobj):
-                                #   colormap, contour_levels, diff_colormap, diff_contour_levels, tiString, tiFontSize, mpl
-                                #   *Any other entries will be ignored.
-                                # NOTE: If we were doing all the plotting here, we could use whatever we want from the provided YAML file.
-
-                                nhfig = make_polar_plot(mseasons[s], oseasons[s], dseasons[s], hemisphere="NH", **vres)
-                                shfig = make_polar_plot(mseasons[s], oseasons[s], dseasons[s], hemisphere="SH", **vres)
-
-                                # Save files
-                                nhfig.savefig(nh_plot_name, bbox_inches='tight', dpi=300)
-                                shfig.savefig(sh_plot_name, bbox_inches='tight', dpi=300)
-
-                                # Close figures to avoid memory issues:
-                                plt.close(nhfig)
-                                plt.close(shfig)
+                                for plot_name in [nh_plot_name, sh_plot_name]:
+
+                                    # If redo_plot set to True: remove old plot, if it already exists:
+                                    if (not redo_plot) and plot_name.is_file():
+                                        continue
+                                    else:
+                                        if plot_name.is_file():
+                                            plot_name.unlink()
+
+                                        #Create new plot:
+                                        # NOTE: send vres as kwarg dictionary.  --> ONLY vres, not the full res
+                                        # This relies on `plot_map_and_save` knowing how to deal with the options
+                                        # currently knows how to handle:
+                                        #   colormap, contour_levels, diff_colormap, diff_contour_levels, tiString, tiFontSize, mpl
+                                        #   *Any other entries will be ignored.
+                                        # NOTE: If we were doing all the plotting here, we could use whatever we want from the provided YAML file.
+                                        
+                                        nhfig = make_polar_plot(mseasons[s], oseasons[s], dseasons[s], hemisphere="NH", **vres)
+                                        shfig = make_polar_plot(mseasons[s], oseasons[s], dseasons[s], hemisphere="SH", **vres)
+                                        
+                                        # Save files
+                                        nhfig.savefig(nh_plot_name, bbox_inches='tight', dpi=300)
+                                        shfig.savefig(sh_plot_name, bbox_inches='tight', dpi=300)
+
+                                        # Close figures to avoid memory issues:
+                                        plt.close(nhfig)
+                                        plt.close(shfig)
```

Comment:
We'll want to make the same change here:

```suggestion           
                                     
                                        #Determine hemisphere to plot based on plot file name:
                                        if plot_name == nh_plot_name:
                                            hemi = "NH"
                                        else:
                                            hemi = "SH"
                                        #End if
                                    
                                        fig = make_polar_plot(mseasons[s], oseasons[s], dseasons[s], hemisphere=hemi, **vres)
                                    
                                        # Save file:
                                        fig.savefig(plot_name, bbox_inches='tight', dpi=300)
                                                                        
                                        # Close figure to avoid memory issues:
                                        plt.close(fig)
```

Reply:
Done